docs/hive-integration/kube-api-getting-started.md

```diff
@@ -0,0 +1,493 @@
+         name: pull-secret  # Use the pull secret name mentioned in Cluster Prerequisites step 2
+   EOF
+```
+##### Result
+
+* At this point, there is no `AgentClusterInstall`, so it won't do much and won't even register the cluster.
+
+###### custom resource conditions:
+```bash
+kubectl -n demo-worker4 get clusterdeployments.hive.openshift.io single-node -o=jsonpath="{.metadata.name}{'\n'}{range .status.conditions[*]}{.type}{'\t'}{.message}{'\n'}"
+```
+```bash
+single-node
+RelocationFailed	no ClusterRelocates match
+Hibernating	Condition Initialized
+AuthenticationFailure	Platform credentials passed authentication check
+```
+
+###### assisted-service log:
+```go
+time="2021-06-28T20:45:07Z" level=info msg="ClusterDeployment Reconcile started" func="github.com/openshift/assisted-service/internal/controller/controllers.(*ClusterDeploymentsReconciler).Reconcile" file="/go/src/github.com/openshift/origin/internal/controller/controllers/clusterdeployments_controller.go:115" cluster_deployment=single-node cluster_deployment_namespace=demo-worker4 go-id=684 request_id=b5022e40-4a1e-4658-ba0a-5159938b7789
+time="2021-06-28T20:45:07Z" level=info msg="AgentClusterInstall does not exist for ClusterDeployment single-node" func="github.com/openshift/assisted-service/internal/controller/controllers.(*ClusterDeploymentsReconciler).Reconcile" file="/go/src/github.com/openshift/origin/internal/controller/controllers/clusterdeployments_controller.go:147" AgentClusterInstall=test-agent-cluster-install cluster_deployment=single-node cluster_deployment_namespace=demo-worker4 go-id=684 request_id=b5022e40-4a1e-4658-ba0a-5159938b7789
+time="2021-06-28T20:45:07Z" level=info msg="ClusterDeployment Reconcile ended" func="github.com/openshift/assisted-service/internal/controller/controllers.(*ClusterDeploymentsReconciler).Reconcile.func1" file="/go/src/github.com/openshift/origin/internal/controller/controllers/clusterdeployments_controller.go:112" cluster_deployment=single-node cluster_deployment_namespace=demo-worker4 go-id=684 request_id=b5022e40-4a1e-4658-ba0a-5159938b7789
+```
+
+#### 2. Create [AgentClusterInstal](README.md#agentclusterinstall)
+
+```yaml
+cat <<EOF | kubectl create -f -
+apiVersion: extensions.hive.openshift.io/v1beta1
+kind: AgentClusterInstall
+metadata:
+  name: test-agent-cluster-install
+  namespace: demo-worker4
+spec:
+  clusterDeploymentRef:
+    name: single-node # Use the clusterDeployment name from the previous step
+  imageSetRef:
+    name: openshift-v4.8.0 # Use the ClusterImageSet name mentioned in Cluster Prerequisites step 3
+  networking:
+    clusterNetwork:
+    - cidr: 10.128.0.0/14
+      hostPrefix: 23
+    machineNetwork:
+    - cidr: 192.168.111.0/24
+    serviceNetwork:
+    - 172.30.0.0/16
+  provisionRequirements:
+    controlPlaneAgents: 1
+ #sshPublicKey: ssh-rsa your-public-key-here (optional)
+EOF
+```
+
+##### Result
+
+* Cluster was now registered in assisted-service backend (PostgreSQL DB).
```

Comment:
Again since this is an implementation detail, perhaps just say "The cluster is fully defined".